### PR TITLE
Update docs file

### PR DESCRIPTION
**Pull request by TawfikYasser**
Here are the updates:
- [x] Update the number of contributors.
- [x] Add `api-server` package.

I hope this be good.

Thanks for let me contribute. @thedavidprice

### CODE REVIEW

**Comment by TawfikYasser** [comment]
Hi @thedavidprice  I hope my work was true !

**Comment by thedavidprice** [comment]
Fixes https://github.com/redwoodjs/redwoodjs.com/issues/536

File: docs/contributing.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ What you want to do not on the roadmap? Well, still go for it! We love spikes an
 
 |Package|Description|
 |:-|:-|
+|[`@redwoodjs/api-server`](https://github.com/redwoodjs/redwood/blob/main/packages/api-server/README.md)|Run a Redwood App using an Express server alternative to a serverless API|
```
Comment (thedavidprice) [inline docs/contributing.md:26]:
```suggestion
|[`@redwoodjs/api-server`](https://github.com/redwoodjs/redwood/blob/main/packages/api-server/README.md)|Run a Redwood app using Express server (alternative to serverless API)|
```

**Comment by thedavidprice** [comment]
Thanks so much, @TawfikYasser Merging now 🚀

**Comment by TawfikYasser** [comment]
You are welcome @thedavidprice It’s a great chance for me, I hope to do more.
Thanks

**Comment by thedavidprice** [comment]
Of course! Excited to have you join and help out. I hope you do more as well -- you are welcome to anytime. But never with any pressure.

**Comment by TawfikYasser** [comment]
Thank you very much.

I have a question, Yesterday I wrote an issue in  [learn.redwoodjs.com](https://github.com/redwoodjs/learn.redwoodjs.com/issues/47) repo, If you can check it, This will be great for me :).

**Comment by thedavidprice** [comment]
I looped in Claire, who's the lead on the project. Please keep me posted if you don't hear back in a few days.

**Comment by TawfikYasser** [comment]
> I looped in Claire, who's the lead on the project. Please keep me posted if you don't hear back in a few days.

Okay, Thank you very much

**Pull request merged**